ledger/simulation/trace.go

```diff
@@ -67,36 +69,74 @@ func makeTxnGroupResult(txgroup []transactions.SignedTxn) TxnGroupResult {
 // ResultLatestVersion is the latest version of the Result struct
 const ResultLatestVersion = uint64(1)
 
+// ResultEvalOverrides contains the limits and parameters during a call to Simulator.Simulate
+type ResultEvalOverrides struct {
+	MaxLogCalls *uint64
+	MaxLogSize  *uint64
+}
+
+// SimulateLogBytesLimit hardcode limit of how much bytes one can log during simulation (with lift-log-limits)
```

Comment:
```suggestion
// SimulateLogBytesLimit hardcode limit of how much bytes one can log in one transaction group during simulation (with lift-log-limits)
```

Reply:
I believe it's during just a single transaction